test/Raven/Tests/ClientTest.php

```diff
@@ -694,6 +701,34 @@ public function testCaptureExceptionHandlesExcludeOption()
         $this->assertEquals(0, count($events));
     }
 
+    /**
+     * @covers Raven_Client::captureException
+     */
+    public function testCaptureExceptionHandlesExcludeSubclassOption()
+    {
+        $client = new Dummy_Raven_Client(array(
+            'exclude' => array('Exception'),
+        ));
+        $ex = new Dummy_Exception();
+        $client->captureException($ex, 'test');
+        $events = $client->getSentEvents();
+        $this->assertEquals(0, count($events));
+    }
+
+    /**
+     * @covers Raven_Client::captureException
+     */
+    public function testCaptureExceptionHandlesExcludeInterfaceOption()
+    {
+        $client = new Dummy_Raven_Client(array(
+            'exclude' => array('Dummy_Exception_Interface'),
+        ));
+        $ex = new Dummy_Exception();
+        $client->captureException($ex, 'test');
+        $events = $client->getSentEvents();
+        $this->assertEquals(0, count($events));
+    }
+
```

Comment:
Great, you added tests upfront! :+1: 
You should add a test about "malformed" strings in the `excepts` option, though; I'm not sure how that would be handled.

Reply:
Well `is_a` just returns `false` when you pass it stuff that isn't a class name,